news/2022-31-07-osulazer-audio-design.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "sound design of osu!(lazer)"
+date: 2022-07-31 00:00:00 +0000
+---
+
+![](/wiki/shared/news/2022-07-31-lazer-sound-design/header.png)
+
+Recently I've been very fascinated with the sound design of osu!(lazer), so much so that I decided to deep dive it and do a write-up about it!
+
+Sound design relates to how any particular sound can make its listener *feel*. The sound itself can be anything from the pop and fizz when opening a can of Coca-Cola, to the continuous rumble of a car engine. In video games, sounds need to match the things players interact with, so for osu! the question is: "What sounds should things like a circle, button, or a toggle have?"
+
+osu!'s new client intends to answer that question with lots of brand new changes, including brand new sound effects to suit the brand new UI. If you are wondering where they come from or how they are made, read along for an interview with [nekodex](https://osu.ppy.sh/users/102), the person behind the sound design of osu!(lazer).
+
+Jamie Taylor is a full-stack engineer born in Australia that joined the osu!team as a web developer in 2015 after peppy asked if he was interested in helping rebuild the osu! website. Jamie has been with the team ever since, working on various osu! related things such as the review system for beatmap modding, the chat system etc., but today he describes himself as a hybrid audio designer/developer and does is not involved as much in the web side of things. Today we will look at Jamie's alias, nekodex, how he developed as an artist and focus on the work he's done with osu!, more specifically, osu!(lazer)'s sound design.
+
+Nekodex as an alias came to be in 2012, as Jamie had musical background while growing up, played with early music software, learned from YouTube tutorials and eventually started messing around in DAW's (digital audio workstations) until music came out. As Jamie was getting serious with music production peppy was preparing osu!stream (an iOS spinoff of osu! that was created in 2010) for the public view, which required original music for the game. As peppy and Jamie are real life friends and peppy knew about his music he got the chance to create some themes and tracks for osu!stream and later on got some of his own songs mapped for osu!stream as well.
```

Comment:
> Today we will look at Jamie's alias, nekodex

there's nothing here about nekodex's name other than the date.. am i missing something?

Reply:
I agree with this, I have reached out once more to nekodex to beef this section up

Reply:
nekodex performed his review and proposed that the alias part is ditched which I did, so resolving this